groups/groups_db.go

```diff
@@ -88,21 +114,32 @@ func FindGroupByName(ctx context.Context, name string, accountID string) (*Servi
 		return nil, false
 	}
 
-	var group ServiceGroup
+	var (
+		group     ServiceGroup
+		createdAt pgtype.Timestamp
+		updatedAt pgtype.Timestamp
+	)
 
-	sqlStatement := `SELECT id, name, account_id, template_id, capacity
-FROM triton.tsg_groups
+	sqlStatement := `
+SELECT id, name, account_id, template_id, capacity, health_check_interval, created_at, updated_at
```

Comment:
I believe, the `health_check_interval` should be removed (we only keep it in the schema, etc.).